gatsby/src/templates/ApiReference/Schema/Content/Content.module.scss

```diff
@@ -0,0 +1,77 @@
+@import '../styles/variables';
+
+.schema__description {
```

Comment:
It seems redundant to prefix with `schema__`, when we have the benefit of css modules preprending the module name to the class name. Was there a reason for this BEM-like approach?

Reply:
I was using BEM conventions before I refactored a really large component into several smaller ones. I'll go through and remove these.